bootstrap/cmd/bootstrap/app/router.go

```diff
@@ -439,18 +468,19 @@ func (r *kfctlRouter) CreateDeployment(ctx context.Context, req kfdefs.KfDef) (*
 
 	// Continue request process in separate thread.
 	go c.CreateDeployment(context.Background(), req)
-	return &req, nil
-}
-
-// GetLatestKfdef returns latest kfdef status.
-func (r *kfctlRouter) GetLatestKfdef(req kfdefs.KfDef) (*kfdefs.KfDef, error) {
-	name, err := k8sName(req.Name, req.Spec.Project)
-	if err != nil {
-		log.Errorf("Could not generate the name; error %v", err)
-		return nil, err
-	}
-	address := fmt.Sprintf("http://%v.%v.svc.cluster.local:80", name, r.namespace)
-	log.Infof("Creating client for %v", address)
-	c, err := NewKfctlClient(address)
-	return c.GetLatestKfdef(req)
+	// TODO
+	return &kfdefs.KfDef{}, nil
 }
+//
+//// GetLatestKfdef returns latest kfdef status.
```

Comment:
Please add a TODO explaining why this is commented out.

The best thing would be to link to an issue.